packages/calling/calling/src/Calling/types.ts

```diff
@@ -0,0 +1,37 @@
+import {CallingEventTypes} from '../Events/types';
+import {ICallHistory} from '../CallHistory/types';
+import {ICallSettings} from '../CallSettings/types';
+import {CallingClientConfig, ICallingClient} from '../CallingClient/types';
+import {IContacts} from '../Contacts/types';
+import {LoggerConfig} from '../Logger/types';
+import {ISDKConnector} from '../SDKConnector/types';
+import {IVoicemail} from '../Voicemail/types';
+import {Eventing} from '../Events/impl';
+
+interface ClientConfig {
+  calling?: boolean;
+  contact?: boolean;
+  history?: boolean;
+  settings?: boolean;
+  voicemail?: boolean;
+}
+
+export interface WebexConfig {
+  access_token: string;
```

Comment:
Are we not going to add any other members in the WebexConfig?
We should be giving the developer the ability to set additional parameters in the webex config right?